build/teamcity/cockroach/ci/tests-aws-linux-arm64/acceptance.sh

```diff
@@ -1,5 +1,46 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -xeuo pipefail
 
-source build/teamcity/cockroach/ci/tests/acceptance.sh
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-support.sh"
+source "$dir/teamcity-bazel-support.sh"  # For run_bazel
+
+if [[ "$(uname -m)" =~ (arm64|aarch64)$ ]]; then
+  export CROSSLINUX_CONFIG="crosslinuxarm"
```

Comment:
In this case, `CROSSLINUX_CONFIG` will always be `crosslinuxarm`, so you could hard-code it. No harm in keeping it the way it is though.